### DISCUSSION TITLE
C plus plus

### DISCUSSION THREAD

**Discussion by hungan123** [community/community#134514] [Programming Help]
### Select Topic Area

Question

### Body

I have learned python very well and now I want to learn c++ can anyone give me the files about c++🤗🤗

**Comment by hamfan524** [comment]
Hello! If you are completely new to the C++ language, how about following this lecture?

[Youtube C++ Tutorial](https://www.youtube.com/watch?v=s0g4ty29Xgg&list=PLBlnK6fEyqRh6isJ01MBnbNpV3ZsktSyS)

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @hungan123 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by srijon57** [comment]
you can see this roadmap https://roadmap.sh/cpp .
if you want tutorial New boston's tutorial finds me better https://www.youtube.com/playlist?list=PLAE85DE8440AA6B83
for book teach yourself c++ herbert schild is good https://archive.org/details/teachyourselfc000schi
